<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>406</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="4">
     <widget class="QPushButton" name="Subtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="MemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Dot">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="MemClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Divide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="MemGet">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #FFFFFF;
	color: black;
	font: 700 20pt &quot;Century Gothic&quot;;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLineEdit" name="History">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #FFFFFF;
	color: black;
	font: 700 20pt &quot;Century Gothic&quot;;
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #FFFFFF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #E0DFDF;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Multiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Percent">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: italic 9pt &quot;Californian FB&quot;;</string>
      </property>
      <property name="text">
       <string>History:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Sqr">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #F29191;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}
QPushButton:pressed {
	background-color: #D48181;
	border-radius: 15px;
	border: 1px solid;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
